src/target.zig

```diff
@@ -684,6 +684,7 @@ pub fn toLLVMTriple(target: std.Target, buf: []u8) []const u8 {
         .watchos => "watchos",
         .driverkit => "driverkit",
         .shadermodel => "shadermodel",
+        .liteos => "liteos",
```

Comment:
I added this in order to make it compile, but https://github.com/Vexu/arocc/pull/510 fixes this.